oops/oops.go

```diff
@@ -24,6 +24,9 @@ func init() {
 // caller's responsibility to ensure that there are not multiple callers in the same program calling
 // `SetPrefixesToShortCircuit` concurrently, as this may cause unexpected behavior where different prefixes are
 // filtered from stacktraces based on the set of values passed by the most recent caller of this function.
+// Note that the file prefix which is evaluated against the set of prefixes provided to this function when an oops
```

Comment:
Nit: Prefer to put a newline between these lines so they're broken up into separate paragraphs in Godocs.